### DISCUSSION TITLE
Fatsti - a Laravel task scheduler with calendar-based management

### DISCUSSION THREAD

**Discussion by a-bashtannik** [HassanZahirnia/laravel-package-ocean#240] [Package Suggestions]
https://github.com/a-bashtannik/fasti 

Developer-friendly, testable job scheduling in one line:

```php
$job = new SendGreetingEmail($user);

Fasti::schedule($job, '2024-12-31 23:59:59'); // Schedule a job for New Year's Eve 2024

// test

Fasti::assertScheduled(SendGreetingEmail::class);

```

Easy to extend by implementing some interfaces. Provides some help commands out of the box. Without redundand resources, a package with only one responsibility: schedule and run your tasks, synchronously or in queue.

**Comment by HassanZahirnia** [comment]
Thanks for the suggestion 😊 I've added the package!
